services/worker-manager/src/api.js

```diff
@@ -918,7 +918,6 @@ builder.declare({
         capacity: worker.capacity || 0,
         providerId: worker.providerId || 'none',
         quarantineUntil: worker.quarantineUntil?.toJSON(),
-        quarantineDetails: worker.quarantineDetails,
```

Comment:
Why was this removed?

Reply:
yeah, that’s where the test failed that I added. So this is a listWorkers endpoint, and I decided not to include this field there
only single getWorker returns it
it was added by accident there initially